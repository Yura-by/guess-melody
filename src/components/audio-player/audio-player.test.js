import React from 'react';
import renderer from 'react-test-renderer';
import AudioPlayer from './audio-player.jsx';

it(`Snapshot test for AudioPlayer`, () => {
  const tree = renderer
    .create(<AudioPlayer
      src="https://web.archive.org/web/20060818144601/http://www.navyband.navy.mil/anthems/ANTHEMS/Belarus.mp3"
      isPlaying={false}
      onPlayButtonClick={() => {}}
    />,
    {
      createNodeMock: () => {
        return {};
      }
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
