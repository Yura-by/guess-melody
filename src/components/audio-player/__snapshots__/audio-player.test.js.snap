// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for AudioPlayer 1`] = `
Array [
  <button
    className="track__button track__button--play"
    disabled={false}
    onClick={[Function]}
    type="button"
  />,
  <div
    className="track__status"
  >
    <div />
  </div>,
]
`;
